activities/Stopwatch.activity/index.html

```diff
@@ -1,41 +1,42 @@
 <!DOCTYPE html>
 <html>
 
-  <head>
-    <meta charset="utf-8" />
-    <title>Stopwatch Activity</title>
-    <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, viewport-fit=cover"/>
-    <link rel="prefetch" type="application/l10n" href="locale.ini">
-    <link rel="stylesheet" media="not screen and (device-width: 1200px) and (device-height: 900px)"
-      href="lib/sugar-web/graphics/css/sugar-96dpi.css">
-    <link rel="stylesheet" media="screen and (device-width: 1200px) and (device-height: 900px)"
-      href="lib/sugar-web/graphics/css/sugar-200dpi.css">
-    <link rel="stylesheet" href="css/bootstrap.min.css">
-    <link rel="stylesheet" href="css/bootstrap-tour.min.css">
-    <link rel="stylesheet" href="css/activity.css">
-    <script>if (typeof module === 'object') {window.module = module; module = undefined;}</script>
-    <script src="lib/jquery.min.js"></script>
-    <script src="lib/bootstrap.min.js"></script>
-    <script src="lib/bootstrap-tour.min.js"></script>
-    <script>if (window.module) module = window.module;</script>
-    <script data-main="js/loader" src="lib/require.js"></script>
-    <script type="text/javascript" src="../../cordova.js"></script>
-  </head>
+<head>
```

Comment:
Do not update indentation/spacing.
Only updated lines should be changed in files.

Reply:
Fixed.